### PR TITLE
Add RSS link in /blog

### PR DESCRIPTION
**Pull request by abhishalya**

### CODE REVIEW

**Comment by abhishalya** [comment]
Looks like this:
image

File: _layout/head.html [hunk 1]
```diff
@@ -28,7 +28,11 @@
 
 {{ispage /blog/*}}
 <div class="container blog-title">
-	{{isdef title}}<h1>{{fill title}}</h1>{{end}}
+  {{isdef title}}<h1>{{fill title}}
+    <a type="application/rss+xml" href="https://julialang.org/feed.xml">
+      <i style="color:#e2802f" class="fa fa-rss-square"></i>
+    </a>
+  </h1>{{end}}
```
Comment (tlienart) [inline _layout/head.html:35]: This will show  on *every* blog post, if that's fine then :+1:

**Comment by tlienart** [comment]
Preview: 
- https://julialang.netlify.com/blog/
- https://julialang.netlify.com/blog/2019/12/yao-differentiable-quantum-programming/

File: _layout/head.html (hunk 1 above)
Comment (abhishalya) [reply _layout/head.html:35]: I think it should occur wherever possible, I thought this would be better than keeping it in navbar.

**Pull request merged**